man/systemd.resource-control.xml

```diff
@@ -412,8 +368,8 @@
           in
           <citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
 
-          <para>This setting is supported only if the unified control group hierarchy is used. Use
-          <varname>BlockIOAccounting=</varname> on systems using the legacy control group hierarchy.</para>
+          <para>This setting replaces <varname>BlockIOAccounting=</varname> and disables <varname>BlockIO</varname>
+          prefixed settings.</para>
```

Comment:
BlockIO *and* StartupBlockIO -prefixed. Or maybe it'd be better to just list them.